Formula/xkbcomp.rb

```diff
@@ -20,9 +21,17 @@ class Xkbcomp < Formula
   depends_on "libxkbfile"
 
   def install
-    system "./configure", *std_configure_args
+    # avoid cellar in xkb data dir
+    destdir = buildpath/"dest"
+    system "./configure", "--prefix=#{HOMEBREW_PREFIX}"
```

Comment:
Won't this make the formula break with `brew unlink`?

Reply:
This will not break with `brew unlink`, `xkbcomp` only contains `man`, `bin` and `lib`, nothing about `share/X11/xkb`